src/main/java/mate/academy/EventManager.java

```diff
@@ -1,19 +1,64 @@
 package mate.academy;
 
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
 public class EventManager {
-    public void registerListener(EventListener listener) {
+    public static final int THREAD_POOL_SIZE = 1;
+    public static final int SHUTDOWN_TIMEOUT_MINUTES = 2;
```

Comment:
The constants THREAD_POOL_SIZE and SHUTDOWN_TIMEOUT_MINUTES are public, which means they can be accessed from anywhere. If these values are not intended to be modified or used outside of this class, consider changing their access modifier to private to encapsulate them within the class.